__tests__/endpoints.test.js

```diff
@@ -51,8 +51,37 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+  test("Should return article that corresponds to a given article id", () => {
+    return request(app)
+      .get('/api/articles/6')
+      .expect(200)
+      .then(({ body }) => {
+          expect(body).toHaveProperty('author');
+          expect(body).toHaveProperty('title');
+          expect(body).toHaveProperty('article_id');
+          expect(body).toHaveProperty('body');
+          expect(body).toHaveProperty('topic');
+          expect(body).toHaveProperty('created_at');
+          expect(body).toHaveProperty('votes');
+          expect(body).toHaveProperty('article_img_url');
+      });
+      });
+});
   
+describe('GET /api/articles/:article_id', () => {
+  test("Should respond with 404 Not found for an  article_id that does not exist", () => {
+    return request(app)
+      .get('/api/articles/76')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not found')
+      });
+      });
+});
 
```

Comment:
Could we add one more test here for an invalid ID. I.e. if we request /api/articles/bananas ?